travis.sh

```diff
@@ -141,8 +142,10 @@ my_travis_wait 60 catkin build --no-status --summarize || exit 1
 # Source the new built workspace
 travis_run source install/setup.bash;
 
-# Only run tests on the current repo's packages
-TEST_PKGS=$(catkin_topological_order $CI_SOURCE_PATH --only-names)
+# Choose which packages to run tests on
+echo "Test blacklist: $TEST_BLACKLIST"
+TEST_PKGS=$(catkin_topological_order src --only-names | grep -Fvxf <(echo "$TEST_BLACKLIST" | tr ' ;,' '\n'))
```

Comment:
Yeah, it was working. Although these changes now break it :p 

I'll prepare another PR from this one with the original `$CI_SOURCE_PATH`.